index.html

```diff
@@ -11,6 +11,22 @@
 
 </head>
 
```

Comment:
For the logo and links in the nav you can put them all together in nav. That way you can apply flexbox and it will affect everything within the nav.